rasa/graph_components/validators/config_validator.py

```diff
@@ -0,0 +1,514 @@
+from __future__ import annotations
```

Comment:
Can we rename the module `default_recipe_validator` then? (or something in this direction)

Reply:
(the test module as well)